ssntp/ssntp.go

```diff
@@ -710,6 +711,9 @@ const defaultServerCert = "/etc/pki/ciao/cert-Server-localhost.pem"
 const defaultClientCert = "/etc/pki/ciao/client.pem"
 const defaultSchedulerCert = "/etc/pki/ciao/cert-Scheduler-localhost.pem"
 
+// Default CIAO certs path
+const ciaoCertsPath = "etc/pki/ciao/"
```

Comment:
@sameo,  It was my bad, it is an absolute path ``/etc/pki/ciao`` , code updated.